flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -435,6 +436,18 @@ class OmpAttributeVisitor : DirectiveAttributeVisitor<llvm::omp::Directive> {
   void Post(const parser::OpenMPAllocatorsConstruct &);
 
   // 2.15.3 Data-Sharing Attribute Clauses
+  void ResolveNames(const parser::OmpObjectList &objList);
+
+  bool Pre(const parser::OmpClause::Inclusive &x) {
+    const auto &objectList{x.v};
+    ResolveNames(objectList);
```

Comment:
Can you add a debug-dump-symbols test?

Reply:
I have added a new flag and dumped the symbols in a new test.